src/api/operations.ts

```diff
@@ -245,7 +245,7 @@ export const drawRectangle = (options: {
 
   // Ensure rx and ry are within bounds
   const rx = Math.max(0, Math.min(options.rx || 0, w / 2));
-  const ry = Math.max(0, Math.min(options.ry || 0, h / 2));
+  const ry = Math.min(0, Math.max(-(options.ry || 0), h / 2));
```

Comment:
This change break the border radius of rectangles when they are drawn by `drawSvg`. It happens because a negative value is used as the height to compensate the Y-axis inversion that is applied by `drawSvgPath`.

On this PR #81 I changed the approach to use a translation instead of passing a negative value.